html.yaml

```diff
@@ -0,0 +1,31 @@
+---
+deliverables:
+- name: "HTML5"
+  context:
+    inputs:
```

Comment:
Here we have multiple input values because I'd like to implement multiple input files. However, because we don't support that yet, it may be better to just convert this into

```yaml
input: "userguide.ditamap"
```

and when support for multiple input files is added, then also add

```yaml
inputs:
- "userguide.ditamap"
```

I don't like if we support multiple ways of defining the same thing, but in this case it may be ok.